tests/microbenchmarks.py

```diff
@@ -49,6 +50,7 @@ def test_booleans(test_case):
             y = (False != False) != False
         """), timed=True)
 
+
 def test_global_var_load(test_case):
     print("Running", "test_global_var_load")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils